src/Aspire.Hosting.Elasticsearch/PublicAPI.Shipped.txt

```diff
@@ -1 +1,11 @@
 #nullable enable
+Aspire.Hosting.ApplicationModel.ElasticsearchResource
```

Comment:
This one hasn't shipped stable yet.

Reply:
Oh, good catch, thought I removed all of the non stable ones.